gradlew

```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env sh
+#!/bin/sh
```

Comment:
Should use `#!/usr/bin/env sh` here. This will point to `sh` in the user's `$PATH` vs using `/bin/sh` which may be different than what is in the `$PATH`.